stats.py

```diff
@@ -658,6 +708,8 @@ def process():
 
         comments = get_comments(revision, sessions)
 
+        review_groups = get_review_groups(sessions)
```

Comment:
We should move this outside the revisions loop, since it only needs to run once, instead of once per each revision. :)